stacker/commands/stacker/info.py

```diff
@@ -0,0 +1,15 @@
+"""Gets information on the CloudFormation stacks based on the given config."""
+
+from .base import StackerCommand
+from ...actions import info
+
+
+class Info(StackerCommand):
```

Comment:
Yeah, that's fine.  If you think there's more info we can provide, lets leave it as Info - doesn't necessarily need to happen in this PR.